### DISCUSSION TITLE
First "Smoke-Test" impressions

### DISCUSSION THREAD

**Discussion by janosch79** [PE5PVB/SI4684-DAB-Receiver#3] [General]
Hello Sjef,
thanks again for this great project!
After build-up the set a few questions and sugesstions come up:
- in my fist attempt, I ordered the wrong LM1117T regulator. A note to keep strict to the 3.3V version would protect some followers and their expesive tuner chips....The maximum rating of the Si4684 keeps strict to 3.6V. For a implementing the set as a "maker project" a protection in form of a 3,6V Zener-Diode on the 3.3V rail might bring more safety here.
- the headamp amp is not working on my first sample. I checked the QFN solder and all supply conditions. The ratings and input signals look good. In the youtube introduction video, you used the cinch output (working fine here as well), but was the hphone-amp verified on your side?  
-  The 2 strongest, local multiplexes don't play on my set. But all weaker and almost DX muxes work fine. Probably a hyper-sensitive problem that overdrives the input-level threshold..? My antenna is a roof areal (Logpeer) directed to the DX origins but still with considerable level on the local ones.
- The FM-Part of the Si4684 is (still) not included in the API. Whilst 95% of the mem is reached after compilation, is there any hope for inclusion, or would this throw the space avilalable in the ESP32 WROOM? 

As a sugestion I attached a language.h with German support. The translation is based on the English original and adapted to more common expressions.

Kind Regards

Jan 

(janosch79(79))

[language.zip](https://github.com/PE5PVB/SI4684-DAB-Receiver/files/15142519/language.zip)

**Comment by PE5PVB** [comment]
Hi,

1. It should be pretty clear. In the schematic the value of the component is: LM1117T-3.3
2. Yes, headphones amplifier should work fine. Ofcourse I tested that. Currently 20 PCB's are build under my supervision, and all headphone amplifiers are working fine.
3. That is strange. Are you using an active antenna? I have a DAB transmitter nearby, no problems with that.
4. Well, the basic ESP32 has 4MB of flash, there is no room left for FM firmware (512kB) and additional code. But... Next project will be a combination of the SI4684 DAB with the TEF for FM.

**Comment by janosch79** [comment]
Hi, 
1... my fault. No question. But my point was more a hint for the future kit to prevent these mistakes.
2.. Ok. Thanks, I got to replace mine as they probably didn't make it under the solder process..
3.. Yes, its an active distribution with an old TV-Multiband-Amp through a 9/20 Satellite-Multiswitch. It is affecting in particular the 3 stongest muxes (11B/9D/9B here). 5C with comparable strength level however is performing well. All weaker ones do fine. Didn't experince something like that with my other DAB receivers.. I can support you with a spectrum plot or demo pics later...
4.. Thanks. Now I can stop making thoughts on that. Or probaly than as an alternative image for the ESP32...

Regards,  
Jan
